### DISCUSSION TITLE
Is there a way to create a subset of paid team members in a Github Free Org?

### DISCUSSION THREAD

**Discussion by kentwongg** [community/community#69217] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

As per the title,

I am part of a Github Free Organization tier, so it does not have branch protection. Is it possible a subset/nested team pay for the GitHub Team tier and we can enable branch protection on our subset repositories within the organization?

Or is this not possible, and that every member within the organization must be on the paid team tier?

Thanks

**Comment by kentwongg** [comment]
bump

**Comment by ayangabryl** [comment, marked as answer]
No
